### PR TITLE
Folders for languages

### PR DESCRIPTION
**Pull request by bruch-alex**
Before this PR we had only 1 property file for each language.

Before:

- strings_en.properties
- strings_de.properties
- ...

After:

- eng/ui.properties
- eng/countries.properties
- eng/products.properties
- deu/ui.properties
- deu/countries.properties
- ...

So i did that and added some spacing between elements and nice looking separator

### CODE REVIEW

**Comment by bruch-alex** [comment]
Screenshot from 2024-11-27 13-12-04

File: src/main/resources/fxml/main.fxml [hunk 1]
```diff
@@ -1,30 +1,39 @@
...
+        <fx:include fx:id="menuBarInclude" source="ui/menu-bar.fxml" />
     </top>
 
     <center>
         <VBox alignment="TOP_CENTER" prefHeight="200.0" prefWidth="100.0" BorderPane.alignment="CENTER">
-<!--            <fx:include fx:id="iconThemeSwitcher" source="ui/themeSwitcher/icon-theme-switcher.fxml"/>-->
-            <fx:include source="ui/bar-chart-view.fxml"/>
+            <!--            <fx:include fx:id="iconThemeSwitcher" source="ui/themeSwitcher/icon-theme-switcher.fxml"/>-->
+            <fx:include source="ui/bar-chart-view.fxml" />
+         <Separator prefWidth="200.0" />
```
Comment (nat-laz) [inline src/main/resources/fxml/main.fxml:22]: they even have components for Styling 👀 noice 
good to know !

File: src/main/java/org/example/globalgoodsindex/core/services/L10N.java [hunk 2]
```diff
@@ -52,21 +52,28 @@ public static ObjectProperty<Locale> localeProperty() {
...
+        ResourceBundle products = ResourceBundle.getBundle("L10n/" + getLocale().getISO3Language() + "/products", getLocale());
+        ResourceBundle ui = ResourceBundle.getBundle("L10n/" + getLocale().getISO3Language() + "/ui", getLocale());
+
+        ArrayList<ResourceBundle> bundles = new ArrayList<>(List.of(
+                countries, products, ui
+        ));
+        for (var bundle : bundles){
+            if (bundle.containsKey(key)) return MessageFormat.format(bundle.getString(key), args);
+        }
+        return null;
```
Comment (nat-laz) [inline src/main/java/org/example/globalgoodsindex/core/services/L10N.java:70]: great staff is going on here 🤩

**Review by nat-laz — APPROVED** [review]
are you doing `refactoring` as a hobby? 

great work, man ! 😻

**Review by nat-laz — APPROVED** [review]

**Review by nat-laz — APPROVED** [review]

**Review by nat-laz — APPROVED** [review]

**Review by nat-laz — APPROVED** [review]

**Pull request merged**